src/app.rs

```diff
@@ -85,7 +85,10 @@ pub fn build_app() -> App<'static, 'static> {
             arg("exec")
                 .long("exec")
                 .short("x")
-                .takes_value(true)
+                .multiple(true)
+                .min_values(1)
+                .allow_hyphen_values(true)
+                .value_terminator(";")
```

Comment:
We should probably keep `;` as terminator here for consistency with `find` but I was shortly wondering if we should maybe change this to something that doesn't need to be escaped on the shell. The `\;` has always looked really awkward to me.

On the other hand, what ever we would choose as an alternative could not be passed to other programs...

Reply:
Yeah, I'm not particularly fond of `;` either. I've more than once been puzzled by odd error messages when I forgot to escape it. `:` would be an alternative that doesn't need escaping (but doesn't convey the same "end of statement" feel).

Reply:
Let's keep `;` for now and I'll keep this in mind. I was also thinking of `:`. Would be great if we (clap) could support both.